spec/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_spec.rb

```diff
@@ -156,18 +156,6 @@
           described_class.create_in_provider(manager.id, params)
         end.to raise_error(RuntimeError, error_msg)
       end
-
-      # TODO:  Determine if we want to have a uniqueness validation to
```

Comment:
FYI:  This file deleted in https://github.com/ManageIQ/manageiq/pull/19383 so this will not be included as part of the next rebase.